modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/plugins/WritingAssistant/types.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+export const ACTION_TYPES_LIST = [
+	'Improve Writing',
+	'Fix Spelling and Grammar',
+	'Translate To',
+	'Make Shorter',
+	'Make Longer',
+	'Generate Based On Title',
+] as const;
```

Comment:
This should probably be an enum, something like 
```js
export enum EActionType {
	IMPROVE_WRITING = 'improveWriting',
    ...
}
```
Current naming implies a list of unlocalized labels.